astropy/table/tests/test_table.py

```diff
@@ -909,6 +909,55 @@ def test_remove_row_preserves_meta(self, table_types):
         assert self.t.dtype == np.dtype([(str('a'), 'int'),
                                          (str('b'), 'int')])
 
+    def test_remove_masked_rows_1(self, table_types):
+        self._setup(table_types)
+        self.t.add_column(self.b)
+        self.t.add_column(self.c)
+        self.t.remove_masked_rows()
```

Comment:
Do:
```
tc = t.copy()
self.t.remove_masked_rows()
assert t.colnames == tc.colnames
assert np.all(t == tc)
```
And add a comment or docstring saying what this is testing (remove_masked_rows with unmasked table).